dependencies.yaml

```diff
@@ -625,7 +633,7 @@ dependencies:
       - output_types: pyproject
         packages:
           - msgpack
-          - &tokenizers tokenizers==0.13.1
+          - &tokenizers tokenizers==0.15.2
```

Comment:
I've bumped tokenizers to 0.15.2 so we can get Python 3.11 binaries.

Reply:
Incompatible with `transformers==4.24.0` :(

Reply:
Bumped transformers to 4.38.1.